packages/utilities/psammead-assets/src/__snapshots__/svgs.test.jsx.snap

```diff
@@ -533,6 +533,35 @@ exports[`kyrgyz SVG should render correctly 1`] = `
 </svg>
 `;
 
+exports[`learningenglish SVG should render correctly 1`] = `
+<svg
+  viewBox="314 24"
+>
+  <g
+    fill-rule="evenodd"
+    stroke="#000"
```

Comment:
Should we have these colours??

Reply:
@greenc05 these are for ensuring it looks good with high contrast mode on PC and Firefox colours being changed.

Reply:
@sareh How will it look good with a background colour of #000?

Reply:
I think this is the same as the other brands, so I'd expect it to work.  As we're all happy can I resolve this conversation?